### PR TITLE
fix(dashboard): attendance weeks only start from Monday correctly

### PR DESCRIPTION
**Pull request by sleidig**
Somehow I am too stupid to get this right. This is the second try after initially tests had failed when running on Sundays ...
Please double-check, @TheSlimvReal

### CODE REVIEW

**Comment by TheSlimvReal** [comment]
As far as I understand the problem, its just a different start of the week day for moment and native date: Moment uses Monday as first day so on Sunday the Monday this week is the previous Monday. In native this would be the following day so the next Monday. So the solution to just use the same calculation method for both works but it kind of makes the test unnecessary as its the same code on both sides.

**Comment by sleidig** [comment]
True, I also saw the parallel of test and implementation. I just want Sunday to be considered the last day of the current week (as is common on almost every "normal" country :sweat_smile: ). For tests maybe a few fixed sample times would be more useful than calculating the expected relative date on the fly.

File: src/app/core/common-components/entity-form/entity-form.service.spec.ts [hunk 1]
```diff
@@ -173,7 +173,7 @@ describe("EntityFormService", () => {
 
     schema.defaultValue = PLACEHOLDERS.NOW;
     form = service.createFormGroup([{ id: "test" }], new Entity());
-    expect(form.get("test")).toHaveValue(new Date());
+    expect(form.get("test").value).toEqual(new Date());
```
Comment (TheSlimvReal) [inline src/app/core/common-components/entity-form/entity-form.service.spec.ts:176]: Whats the difference here?

Comment (sleidig) [reply src/app/core/common-components/entity-form/entity-form.service.spec.ts:176]: the test sometimes failed although the date was exactly the same - I guess if MockDate was used before, it's not reference identical anymore?

**Review by TheSlimvReal — APPROVED** [review]

**Pull request merged**